### ISSUE TITLE
Clean theme table of contents not scrolling

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4264]
**Describe the bug**
In Clean on wider windows, the table of contents for an article is displayed to the right of the the text.  This remains static as the text scrolls.  When the window is shorter than the height of the table of contents, it still remains static until one scrolls to the bottom of the page, this can mean that for example, the References, often the last item is not displayed on screen until one reaches the footer.

**To Reproduce**
1. Go to [example article](https://politicalphilosophyjournal.org/article/id/15481/) with a wide, but short window.
2. Scroll down the page

**Expected behavior**
Either - when the table of contents cannot fit on the height of the screen it needs to be displayed another way, or it should be scrollable independent of the main contents.

**Screenshots**
<img width="1290" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/9b2408a9-ad3a-420a-a344-0a7e7dd00cc2">

The table of contents is static through the whole page scroll right up to this point with the last 5 items hidden, then it finally scrolls for the footer:
<img width="1295" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/09e43916-fc73-4e02-b6ad-1e2aa2b1cf49">

## Front-end Issues
If the issue is front-end specific please add the following details:

**Desktop (please complete the following information):**
 - Mac Chrome Version 125.0.6422.142
 - Same behaviour observed on Safari 17.5

**Smartphone (please complete the following information):**
Not expected, as this is about the wider window location for table of contents.

**Additional context**

Discovered during #4260 because this bug is made worse when viewing the page at sizes required in WCAG 1.4.12, since the change in size can lengthen the table of contents, and thus trigger this bug on windows that otherwise would have had enough space to display the full table.

**Comment by StephDriver** [comment]
branch has been merged into master
